### DISCUSSION TITLE
Getting error =backupstoragelocations.velero.io \ "default\" not found (duplicate)

### DISCUSSION THREAD

**Discussion by tariq1278** [vmware-tanzu/velero#3317] [Community support Q&A] [answered]
Hello Everyone,.I have setup velero on aks azure but getting error =backupstoragelocations.velero.io \ "default\" not found

**Comment by carlisia** [comment, marked as answer]
Hey, this is a duplicate of https://github.com/vmware-tanzu/velero/discussions/3319.
